Makefile

```diff
@@ -190,7 +190,7 @@ FLEXDLL_VS_VERSION_INFO = $(subst .,$(COMMA),$(FLEXDLL_FULL_VERSION))
 
 RC_FLAGS = \
   /d FLEXDLL_VS_VERSION_INFO=$(FLEXDLL_VS_VERSION_INFO) \
-  /d FLEXDLL_FULL_VERSION="$(FLEXDLL_FULL_VERSION)"
+  /d FLEXDLL_FULL_VERSION=\"$(FLEXDLL_FULL_VERSION)\"
```

Comment:
Is this _definitely_ not breaking MSVC builds?

Reply:
We checked with Resource Hacker that when built with MS `rc`, the strings are still correct. llvm-rc cannot process the file without the quotes.